FISH-752 Adds section on aggregated history and the added widget setting (#83)
payara/Payara-Enterprise-Documentation: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc

NOTE: To remove only the monitoring data collection from the server without disabling the monitoring service itself delete the `monitoring-console-core.jar` from the `/glassfish/modules` folder of your installation(s) and restart the server.

[[monitoring-console-history]]
== Enabling Data Aggregation (History)
By default InSight has a sliding window of data per metric of usually one minute length which contains 1 point per second.
This allows typical "real-time" monitoring.

> This points represent the raw data as it is probed/collected.

Reviewer: ```suggestion
These points represent the raw data as it is probed/collected.
```

Without a history points that get "pushed" out of the sliding window are "forgotten".
Since Payara Enterprise 5.23.1 a history can be enabled which aggregates the points per seconds to 1 aggregated point per minute, the minutes to 1 point per hour, the hours to 1 point per day.
These aggregations equally have sliding windows so they hold the recent hour, day or month worth of data.
The longer points are in the past the less points per fixed time-frame there are.
In the most recent minute there are the raw (not aggregated) points per second.
In the most recent hour there is 1 point per minute from aggregating the seconds within the last minute.
In the most recent day there is 1 point per hour from aggregating the minutes within the last hour.
In the most recent month there is 1 point per day from aggregating the hours within the last day.

The aggregation of points must be enabled using the following asadmin command:

		asadmin> set-monitoring-console-configuration --history-enabled=true

> This command starts the aggregation which starts to grow as time passes.

Reviewer: ```suggestion
This command starts the aggregation which increases over time.
```

The aggregated data is only stored in memory.
Widgets can chose to show this data using the _Extras_ - _History_ setting.

When aggregation of points is disabled again using the following asadmin command

		asadmin> set-monitoring-console-configuration --history-enabled=true

all aggregated data is discarded and cannot be restored.
Equally history is lost when the server is restarted.

The aggregated history feature is designed to keep memory usage fairly low.
1000 metrics require in the range of single digit MBs of memory.

> So in most environments the extra memory and CPU effort for the history should not be an issue.

Reviewer: ```suggestion
In typical environments the extra memory and CPU effort for the history should not be an issue.
```

> In the future we might look into persisting the aggregated history for further features.

Author: ```suggestion
```

> In the future we might look into persisting the aggregated history for further features.

Reviewer: `In the future we might look into persisting the aggregated history for further features.`

I don't like the idea of setting up expectation.  Docs should describe existing functionality _only_, not allude to something that might appear in the future.
